schema.json

```diff
@@ -0,0 +1,496 @@
+            "pattern": "^https://bitrise-steplib-collection.s3.amazonaws.com/steps$",
+            "examples": [
+                "https://bitrise-steplib-collection.s3.amazonaws.com/steps"
+            ]
+        },
+        "steps": {
+            "type": "object",
+            "propertyNames": {
+                "minLength": 1
+            },
+            "additionalProperties": {
+                "type": "object",
+                "required": [
+                    "info",
+                    "latest_version_number",
+                    "versions"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "info": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "properties": {
+                            "removal_date": {
+                                "type": "string"
+                            },
+                            "deprecate_notes": {
+                                "type": "string"
+                            },
+                            "asset_urls": {
+                                "type": "object",
+                                "additionalProperties": false,
+                                "maxProperties": 2,
+                                "properties": {
+                                    "icon.svg": {
+                                        "type": "string"
+                                    },
+                                    "icon.png": {
+                                        "type": "string"
+                                    }
+                                }
+                            },
+                            "maintainer": {
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "latest_version_number": {
+                        "type": "string",
+                        "pattern": "^\\d+.\\d+.\\d+$",
+                        "examples": [
+                            "1.0.0"
+                        ]
+                    },
+                    "versions": {
+                        "type": "object",
+                        "propertyNames": {
+                            "pattern": "^\\d+.\\d+.\\d+$"
+                        },
+                        "additionalProperties": {
+                            "type": "object",
+                            "additionalProperties": false,
+                            "required": [
+                                "title",
+                                "summary",
+                                "website",
+                                "source_code_url",
+                                "support_url",
+                                "published_at",
+                                "source"
+                            ],
+                            "properties": {
```

Comment:
please validate that the user facing strings contains only an accepted charset

Reply:
What do you mean? What do you suggest to test?